app/assets/javascripts/spinner.js.erb

```diff
@@ -0,0 +1,7 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Why put this in `app/assets` instead of `app/javascripts`?

Reply:
Probably in order to use ERB, but this is a bit of an anti-pattern compared to most of our JS. Perhaps instead of having the inline erb/html in the js, that can be moved to a slim template and be hidden by default. Then js is only responsible for showing/hiding the necessary html and can be handled via webpack like the rest of the js.

Reply:
Template could work. Or CSS. I just needed to get the path to the image and ERB seemed to simplest way for me in the moment, but definitely not the cleanest.

Reply:
Webpacker supports ERB files. Here is how to do it:

- Run `bundle exec rake webpacker:install:erb`
- Move the file to `app/javascripts`
- Add `.erb` to the `extensions` list in `webpacker.yml`

Reply:
If that doesn't work out of the box, try defining `<% helpers = ActionController::Base.helpers %>` and then refer to the image like this:
```
helpers.asset_path('spinner@2x.gif')
```